packages/components/__mocks__/month-calendar.js

```diff
@@ -13,46 +13,46 @@ class MonthCalendar extends React.Component {
 				<tbody>
 					<tr>
 						<td>
-							<button onKeyDown={event => onKeyDown(event, this.calendarRef, 0)}>JAN</button>
+							<button name='0' onKeyDown={event => onKeyDown(event, this.calendarRef, 0)}>JAN</button>
```

Comment:
This is not valid jsx, please use double quotes

Reply:
fixed